build/lib/src/asset/writer.dart

```diff
@@ -4,24 +4,33 @@
 import 'dart:async';
 import 'dart:convert';
 
-import 'asset.dart';
+import 'id.dart';
 
+/// Abstract class that can write [Asset]s.
```

Comment:
[nit] This comment isn't adding much information over the signature. We get a link to [Asset] but I don't think it's worth the comment